docs/source/backends/zerotier.rst

```diff
@@ -337,9 +332,19 @@ Will be rendered as:
     package zerotier
 
     config zerotier 'ow_zt'
+        option config_path '/etc/openwisp/zerotier'
+        option copy_config_path '1'
         option enabled '1'
         list join '9536600adf654321'
-        option secret '{{zt_identity_secret}}'
+        option secret '{{secret}}'
+
+    # ---------- files ---------- #
+
+    # path: /etc/ow_zerotier/devicemap
```

Comment:
why is this `/etc/ow_zerotier/devicemap` and above is `/etc/openwisp/zerotier`?

Reply:
Fixed, thanks!